js/main.js

```diff
@@ -0,0 +1,47 @@
+// Target open and close triggers in html
+
+const openEls = document.querySelectorAll('[data-open]');
+const closeEls = document.querySelectorAll('[data-close]');
+const openElems = Array.from(openEls);
+const closeElems = Array.from(closeEls);
+const isVisible = 'is-visible';
+const navLinks = document.querySelectorAll('.web-header-link');
+
+// Open modal
+openElems.forEach((e) => {
+  e.addEventListener('click', function () {
+    const modalId = this.dataset.open;
+    document.getElementById(modalId).classList.add(isVisible);
+    document.body.style.overflow = 'hidden';
+  });
+});
```

Comment:
- [ ] Kindly remove the `border-radius` at the top part of your popup navigation bar. The Figma template has it only at the bottom. You can achieve it like this: `border-radius:  same-value same-value;` or `border-radius: 0px 0px same-value same value;`.  Training yourself to keep to templates will help you when you start working in the outside world

|Your Design|
|------------|
| ![image](https://user-images.githubusercontent.com/62938735/189644253-cf85ced9-e9b3-4c9c-a62d-e79c7b4178ae.png) |